components/edit-collective/sections/virtual-cards/VirtualCards.js

```diff
@@ -170,7 +185,16 @@ const VirtualCards = props => {
       </Box>
       <Grid mt={4} gridTemplateColumns={['100%', '366px 366px']} gridGap="32px 24px">
         {data.account.virtualCards.nodes.map(vc => (
-          <VirtualCard key={vc.id} {...vc} />
+          <VirtualCard
+            canEditVirtualCard={
+              props.collective.type === CollectiveType.USER &&
+              vc.provider === 'STRIPE' &&
+              (vc.data.state === 'OPEN' || vc.data.status === 'active')
```

Comment:
Why are we doing a OR here?

Looking at if the case is paused is not good enough?

Reply:
That is correct. Having `vc.data.status === 'active'` seems to be enough. I think I copied this condition from elsewhere but testing indicates that `active` status is all we need to see whether the card is already paused or not.